app/models/user.rb

```diff
@@ -56,7 +56,7 @@ def organisation_valid?
   end
 
   def update_user_forms
-    if role_previously_changed?(from: :trial)
+    if role_previously_changed?(from: :trial, to: :editor) || role_previously_changed?(from: :trial, to: :super_admin)
```

Comment:
I'm a bit nervous about this, because when we add other roles (such as `:admin`) we'll need to update this test. Can we instead make it `&& !` the transitions we don't care about?

Reply:
I don't _think_ there would be any valid transitions that would be excluded at the moment if we did that. By explicitly including rather than excluding transitions we implicitly exclude invalid transitions like the role being set to nil, or any other garbage that isn't in the roles enum. This should never happen because validation, but I don't think there's any harm in being defensive here as well.